csvdata/csvdata.go

```diff
@@ -0,0 +1,76 @@
+package csvdata
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/config"
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/model"
+	"github.com/wcharczuk/go-chart"
+)
+
+func ExtractFromCSV(requestedDate string) (records model.CryptoValueRecords, minValue, maxValue float64) {
```

Comment:
Document all the methods that needed like:

// Method - it returns the csv file